src/main/resources/templates/db_scripts/renameModesOfIssuance.sql

```diff
@@ -0,0 +1,39 @@
+START TRANSACTION;
```

Comment:
Hi @julianladisch, @adamdickmeiss, @skoczko, do we need migration scripts for reference data? 

Are the reference data being updated/reloaded every time the module installed/upgraded?

Reply:
Personally, I don't think we can assume that the reference records will be reloaded during every upgrade, as it might overwrite changes made by users of the tenant.

Alas, automatically migrating the existing record has similar issues.

Reply:
Ok, do you think we should have migration scripts anyway?

> Alas, automatically migrating the existing record has similar issues.

Partially, I think the script will be executed only once and when module version will be updated, then it should not.

Reply:
> Ok, do you think we should have migration scripts anyway?
> 
> > Alas, automatically migrating the existing record has similar issues.
> 
> Partially, I think the script will be executed only once and when module version will be updated, then it should not.

That sole update could still overwrite changes that a tenant has made.

I've raised questions about this on the JIRA issue. I think we need decisions about that area before we can decide if we should provide an automated data upgrade script.

Reply:
Based upon [Charlotte's comment](https://issues.folio.org/browse/MODINVSTOR-431?focusedCommentId=70032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-70032) on the story, this work should include a data upgrade script that always overwrites the name of these records, whether they have changed or not. Meaning we should only match on ID.

Reply:
Changed